src/EPR.Payment.Service/Validations/RegistrationFees/RegulatorDtoValidator.cs

```diff
@@ -8,10 +9,8 @@ public class RegulatorDtoValidator : AbstractValidator<RegulatorDto>
         public RegulatorDtoValidator()
         {
             RuleFor(x => x.Regulator)
-                .NotEmpty().WithMessage("Regulator is required.")
-                .Must(x => !string.IsNullOrWhiteSpace(x)).WithMessage("Regulator is required.")
-                .Must(x => x?.ToUpper() == x).WithMessage("Regulator must be in uppercase.")
-                .Must(RegulatorValidationHelper.IsValidRegulator).WithMessage("Invalid regulator parameter.");
+                .NotEmpty().WithMessage(ValidationMessages.RegulatorRequired)
```

Comment:
use Cascade(CascadeMode.Stop) to stop subsequent validations if a validation fails

Reply:
they want to see all messages like <Regulator is required.; Invalid Regulator.>